sgrpg.js

```diff
@@ -41,7 +44,24 @@ Hooks.once("init", function(){
     Items.registerSheet("sgrpg", SGItemSheet, {makeDefault: true});
 
     Actors.unregisterSheet("core", ActorSheet);
-    Actors.registerSheet("sgrpg", SGActorSheet, {makeDefault: true});
+    Actors.registerSheet("sgrpg", SGActorSheet, { makeDefault: true });
+
+
+    game.settings.register("sgrpg", "campaignTension", {
+        name: "Campaign Tension Level",
+        hint: "Set the campaign's base Tension level and the Tension Die.",
+        scope: "world",
+        type: String,
+        choices: {
+            "d4": "Comedic (d4)",
+            "d6": "Standard (d6)",
+            "d8": "Growing (d8)",
+            "d10": "Dire (d10)",
+            "d12": "All is Lost (d12)",
+        },
```

Comment:
This is object mentioned in comment in https://github.com/sadovsf/StargateRPGSystem/pull/13/files#r705242169